test/service/test_response.py

```diff
@@ -1251,8 +1258,23 @@ class TestResponseSummary(WebServiceTestCase):
     maxDiff = None
     bundles = WebServiceTestCase.bundles + [
         ('dcccb551-4766-4210-966c-f9ee25d19190', '2018-10-18T204655.866661Z'),
+        ('94f2ba52-30c8-4de0-a78e-f95a3f8deb9c', '2019-04-03T103426.471000Z')  # an imaging bundle
     ]
 
+    def test_summary_response(self):
```

Comment:
I might be helpful to explain some of the magic numbers in the assertions below. Like, why is cell count for brain zero?